### PR TITLE
For r600

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: meta-aos-rcar-gen3-domd/recipes-aos/aos-vis/aos-vis_git.bbappend [hunk 1]
```diff
@@ -1,14 +1,14 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-SRC_URI_append = " \
+SRC_URI_append = "\
```
Comment (al1img) [inline meta-aos-rcar-gen3-domd/recipes-aos/aos-vis/aos-vis_git.bbappend:3]: Not relevant change. Keep space after ". Most yocto recipes keep it.

File: meta-aos-rcar-gen3-domd/recipes-aos/aos-vis/aos-vis_git.bbappend [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
     file://aos_vis.cfg \
     file://aos-vis.service \
 "
 
-AOS_VIS_PLUGINS = " \
+AOS_VIS_PLUGINS ?= "\
```
Comment (al1img) [inline meta-aos-rcar-gen3-domd/recipes-aos/aos-vis/aos-vis_git.bbappend:8]: ditto. No need for '?'.

File: meta-aos-rcar-gen3-domd/recipes-aos/aos-vis/aos-vis_git.bbappend [hunk 3]
```diff
@@ -50,6 +50,8 @@ python do_configure_adapters() {
 }
 
 do_install_append() {
+    install -d ${D}/${localstatedir}/aos/vis
```
Comment (al1img) [inline meta-aos-rcar-gen3-domd/recipes-aos/aos-vis/aos-vis_git.bbappend:53]: VIS now creates folder itself.

File: meta-aos-rcar-gen3-domf/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend [hunk 4]
```diff
@@ -1,20 +1,17 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-SRC_URI = "git://git@github.com/al1img/aos_iamanager.git;branch=gen4_integration;protocol=ssh"
-SRCREV = "${AUTOREV}"
-
-SRC_URI_append = " \
+SRC_URI_append = "\
```
Comment (al1img) [inline meta-aos-rcar-gen3-domf/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend:3]: Keep space after "

File: meta-aos-rcar-gen3-domf/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend [hunk 5]
```diff
@@ -1,20 +1,17 @@
...
     file://aos_iamanager.cfg \
     file://aos-iamanager.service \
     file://domd_provfinish.sh \
 "
 
-AOS_IAM_CERT_MODULES = " \
+AOS_IAM_CERT_MODULES = "\
```
Comment (al1img) [inline meta-aos-rcar-gen3-domf/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend:9]: ditto

File: meta-aos-rcar-gen3-domf/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend [hunk 6]
```diff
@@ -1,20 +1,17 @@
...
     certhandler/modules/pkcs11module \
     certhandler/modules/swmodule \
 "
 
-AOS_IAM_IDENT_MODULES = " \
+AOS_IAM_IDENT_MODULES = "\
```
Comment (al1img) [inline meta-aos-rcar-gen3-domf/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend:14]: ditto

File: meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend [hunk 7]
```diff
@@ -10,28 +10,16 @@ inherit systemd
 
 SYSTEMD_SERVICE_${PN} = "aos-servicemanager.service"
 
-RDEPENDS_${PN} += " \
+RDEPENDS_${PN} += "\
```
Comment (al1img) [inline meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend:13]: ditto

File: meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend [hunk 8]
```diff
@@ -10,28 +10,16 @@ inherit systemd
...
     aos-rootca \
 "
 
 # Base layer for services
-RDEPENDS_${PN} += " \
+RDEPENDS_${PN} += "\
```
Comment (al1img) [inline meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend:18]: ditto

File: meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg [hunk 9]
```diff
@@ -18,5 +19,8 @@
     "migration": {
         "migrationPath" : "/usr/share/aos/sm/migration",
         "mergedMigrationPath" : "/var/aos/workdirs/sm/migration"
+    },
+    "journalAlerts": {
+        "filter": ["50-udev-default.rules"]
```
Comment (al1img) [inline meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg:24]: Is it relevant for gen3?

File: meta-aos-rcar-gen3-domf/recipes-core/images/core-image-tiny-initramfs.bbappend [hunk 10]
```diff
@@ -3,9 +3,7 @@ COMPATIBLE_HOST = "aarch64.*-linux"
 IMAGE_NAME = "initramfs-domf"
 
 PACKAGE_INSTALL = " \
-    initramfs-module-rundir \
```
Comment (al1img) [inline meta-aos-rcar-gen3-domf/recipes-core/images/core-image-tiny-initramfs.bbappend:6]: Thes change not needed.

File: meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg (hunk 9 above)
Comment (Eabramov84) [reply meta-aos-rcar-gen3-domf/recipes-aos/aos-servicemanager/files/aos_servicemanager.cfg:24]: removed

**Pull request merged**